behaviortree_ros2/src/tree_execution_server.cpp

```diff
@@ -270,9 +278,9 @@ const std::string& TreeExecutionServer::goalPayload() const
   return p_->payload;
 }
 
-BT::Tree* TreeExecutionServer::tree()
+const BT::Tree& TreeExecutionServer::tree() const
 {
-  return p_->tree ? p_->tree.get() : nullptr;
+  return p_->tree;
```

Comment:
Is there any chance of the user accessing the tree and it being outdated or empty? It doesn't seem like there is.

Reply:
Technically an empty Tree can be recognized.
But in general I don't believe this can happen, this is the reason why I made the change